Sources/core/objects/Human.cpp

```diff
@@ -503,6 +792,39 @@ void Human::Bump(IdPtr<IMovable> item)
     IMovable::Bump(item);
 }
 
+void Human::RotationAction(IdPtr<IOnMapBase> item)
+{
+    if(IdPtr<IMovable> movable = item)
+    {
+        if (!movable->anchored)
+        {
+            if (IdPtr<Projectile> projectile = movable)
+            {
+                return;
+            }
+            else
+            {
+                movable->Rotate(movable->GetDir() + 1 % 4);
```

Comment:
It is just `movable->Rotate(movable->GetDir() + 1);` - it will probably cause the crash of the game. Brackets are missing.